scripts/stf/filters.json

```diff
@@ -0,0 +1,43 @@
+[
+    "CustomLabel.npe03__RecurringDonationInstallmentPeriodMonthly",
+    "CustomLabel.npe03__RecurringDonationInstallmentPeriodQuarterly",
+    "CustomLabel.npe03__RecurringDonationInstallmentPeriodWeekly",
+    "CustomLabel.npe03__RecurringDonationInstallmentPeriodYearly",
+    "CustomLabel.npe03__RecurringDonationMultiplyValue",
+    "PicklistValue.Account.npo02__SYSTEM_CUSTOM_NAMING.Name",
+    "PicklistValue.Contact.npe01__PreferredPhone.Home",
+    "PicklistValue.Contact.npe01__PreferredPhone.Mobile",
+    "PicklistValue.Contact.npe01__PreferredPhone.Other",
+    "PicklistValue.Contact.npe01__PreferredPhone.Work",
+    "PicklistValue.Contact.npe01__Preferred_Email.Alternate",
+    "PicklistValue.Contact.npe01__Preferred_Email.Personal",
+    "PicklistValue.Contact.npe01__Preferred_Email.Work",
+    "PicklistValue.Contact.npe01__SystemAccountProcessor.One-to-One",
+    "PicklistValue.Contact.npo02__SystemHouseholdProcessor.All Individual Contacts",
+    "PicklistValue.Contact.npo02__SystemHouseholdProcessor.All New or Edited Contacts",
+    "PicklistValue.Contact.npo02__SystemHouseholdProcessor.No Contacts",
+    "PicklistValue.Opportunity.npsp__Matching_Gift_Status.Potential",
```

Comment:
why did you exclude these.  the 20 places in the code that looks at status use the hardcoded strings, which is what should be internally stored for the picklist values.  so having the user translate the localized version of the picklist value should be fine since the non-translated value is what gets saved for the picklist.  If I'm wrong, then my whole understanding of picklist values is off!